### PR TITLE
A11y docs in storybook

### PR DESCRIPTION
**Pull request by katarzynatobis**
Move accessibility documentation from confluence to storybook

Ready:
- Avatar
- FileHandler
- FlashMessage
- HomeButton
- Icon
- Logo
- MathSymbol
- SubjectIcon
- Accordion
- Dialog
- Rating

Missing:
- Link

Screenshot 2022-02-04 at 13 50 55

### CODE REVIEW

File: .storybook/addon-a11yDocs/register.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <DocsPageWrapper>
+      <Description markdown={value}></Description>
+    </DocsPageWrapper>    
+  );
+};
+
+addons.register('a11ydocs', () => {
+  addons.add('a11ydocs/tab', {
+    type: types.TAB,
+    title: 'Accessibility Docs',
```
Comment (coderitual) [inline .storybook/addon-a11yDocs/register.js:19]: Maybe just "Accessibility"?

Comment (katarzynatobis) [reply .storybook/addon-a11yDocs/register.js:19]: We already have "Accessibility" panel, this can be confusing

**Comment by domidomi** [comment]
@coderitual @katarzynatobis what do you think about componentizing some parts of these docs? For example, there is a Pattern-Comment-Status table which I think would be a great candidate

**Comment by coderitual** [comment]
@domidomi let's discuss tomorrow on standup

**Review by coderitual — APPROVED** [review]

**Pull request merged**